resources/systemd/openhab2.service

```diff
@@ -6,13 +6,17 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
-EnvironmentFile=/etc/default/openhab2
-EnvironmentFile=/etc/default/openhab2_dirs
+Environment=OPENHAB_HOME=/usr/share/openhab2
+Environment=OPENHAB_CONF=/etc/openhab2
+Environment=OPENHAB_RUNTIME=/usr/share/openhab2/runtime
+Environment=OPENHAB_USERDATA=/var/lib/openhab2
+Environment=OPENHAB_LOGDIR=/var/log/openhab2
+Environment=OPENHAB_USER=openhab
+Environment=OPENHAB_GROUP=openhab
+EnvironmentFile=-/etc/default/openhab2
```

Comment:
Good call with the `-`

I'm starting to wonder if all of the above variables should be changeable by the end user (which is the purpose of these lines)